dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -74,45 +74,73 @@ class ScriptSeedTest < ActiveSupport::TestCase
       ScriptSeed.seed_from_json(ScriptSeed.serialize_seeding_json(script))
 
       assert_equal counts_before, get_counts
-      script_after_seed = Script.find_by!(name: script.name)
+      script_after_seed = Script.with_seed_models.find_by!(name: script.name)
       assert_script_trees_equal(script, script_after_seed)
       assert_equal script.script_levels.map(&:id), script_after_seed.script_levels.map(&:id)
     end
 
+    # The following tests ensure that the sequence of serializing and then
+    # seeding produce the right result via the following strategy:
+    #
+    # 1. create script object via create_script_tree. This represents the
+    # initial state of the script, before modifications.
+    #
+    # 2. get_script_and_json_with_change_and_rollback copies and modifies the
+    # script and serializes it to json, and then restores the DB to its original
+    # state via database transaction rollback. It then returns the modified json
+    # the modified script_with_changes, loading script associations into memory
```

Comment:
Something seems a little off in this sentence "the modified json the modified script_with_changes"

Reply:
I think I left out a word. does this sound better?

It then returns the modified json **and** the modified script_with_changes, loading script associations into memory via with_seed_models.